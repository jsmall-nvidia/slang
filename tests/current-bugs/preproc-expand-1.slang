//DIAGNOSTIC_TEST:SIMPLE:-E

// NOTE! This test should *fail*, if preprocessor is working correctly!

// Should produce: Hi
// Slang produces: C ( Hi ) 

#define OPEN (
#define CLOSE )

#define C(x) x

#define B(x) x

B(C OPEN Hi CLOSE)

