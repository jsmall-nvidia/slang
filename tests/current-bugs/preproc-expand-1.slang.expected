result code = 0
standard error = {
}
standard output = {
C ( Hi ) 
}
